doc/api_assets/style.css

```diff
@@ -360,7 +372,7 @@ span.type {
   margin-top: 1.25em;
   background: #f2f5f0;
   margin-bottom: 0;
-  padding-bottom: 2.5em;
+  padding-bottom: 5em;
```

Comment:
I'm not a big fan of how large this padding is, could we do with something smaller, maybe `2em`?